src/Program.cs

```diff
@@ -68,48 +69,123 @@ public class Execute
+            new Arrangement().prepareForStart(Path, configPath, templatePath, out ConfigWithChanges, out dirs);
             //search all files in Directory 
             foreach (var file in dirs)
             {
                 AC = new InputToACObject().parseinobject(new StreamReader(file)); //parses current configuration into the AC object
-                if (customer.HasValue())
-                {
-
-                }
                 if (ConfigWithChanges.configureNetwork != null)
                 {
-                    AC = exe.replaceitem(AC, ConfigWithChanges.configureNetwork.networkdev, "networkdev");
-                    AC = exe.replaceitem(AC, ConfigWithChanges.configureNetwork.interfacenetworkif, "interfacenetworkif");
+                    AC = exe.replaceitem(AC, ConfigWithChanges.configureNetwork.networkdev, "networkdev", country, customer);
+                    AC = exe.replaceitem(AC, ConfigWithChanges.configureNetwork.interfacenetworkif, "interfacenetworkif", country, customer);
                 }
                 if (ConfigWithChanges.configureviop != null)
                 {
-                    AC = exe.replaceitem(AC, ConfigWithChanges.configureviop.proxyset, "proxyset");
-                    AC = exe.replaceitem(AC, ConfigWithChanges.configureviop.proxyip, "proxyip"); //replaces the wanted details
+                    AC = exe.replaceitem(AC, ConfigWithChanges.configureviop.proxyset, "proxyset", country, customer);
+                    AC = exe.replaceitem(AC, ConfigWithChanges.configureviop.proxyip, "proxyip", country, customer); //replaces the wanted details
                 }
                 var objList = obj.objectToList(AC);
                 obj.writeOutput(objList, file); //output
             }
         }
-        protected void change(dynamic i, dynamic item) //replaces the Item
+        protected void change(dynamic i, dynamic item, CommandOption country, CommandOption customer) //replaces the Item
         {
             foreach (var propertyInfo in item.GetType().GetProperties())
             {
                 var value = propertyInfo.GetValue(item);
-                if (value != null)
+                if (value == null)
+                {
+                    if (searchKürzel(propertyInfo.Name))
+                    {
+                        swapKürzel(i, item, country, customer, propertyInfo);
+                    }
+                    continue;
+                }else
+                { 
+                if (searchKürzel(propertyInfo.Name))
+                {
+                    swapKürzel(i, item, country, customer, propertyInfo);
+                }
+                else
                 {
                     i.GetType().GetProperty(propertyInfo.Name).SetValue(i, value);
                 }
+                }
+            }
+        }
+        protected void swapKürzel(dynamic oldconfig, dynamic item, CommandOption country, CommandOption customer, PropertyInfo propertyInfo)
```

Comment:
in the English language, there is no `ü` 🤷‍♀️ maybe you should use a full English name.

Reply:
maybe you check your code base for all `ü`, `ö`, `ä` and the upper case equivalents